pandas/core/arrays/boolean.py

```diff
@@ -755,9 +755,8 @@ def logical_method(self, other):
 
             if other_is_scalar and not (other is libmissing.NA or lib.is_bool(other)):
                 raise TypeError(
-                    "'other' should be pandas.NA or a bool. Got {} instead.".format(
-                        type(other).__name__
-                    )
+                    f"'other' should be pandas.NA or a bool.\
```

Comment:
We avoid backslashes in general, you can close the string here, and start it again in the next line, and they'll concatenate. Just leave a space at the end of this first string, and just make the second a f-string.

Reply:
Good to know, done!